RELEASENOTES.md

```diff
@@ -2,3 +2,6 @@
 - Write warnings to log and console if GitHub is experiencing an availability incident.
 - Improve the error thrown when you have insufficient permissions for the target GitHub organization to explicitly mention the relevant organization
 - Write log output prior to making API calls in wait-for-migration commands
+- Fix the `Multiple repositories found in archive` error by adding migration support for fork repositories. This is accomplished by passing the full
+  Bitbucket Server repository URL to GEI when starting the migration. This URL is constructed using the `--bbs-server-url`, `--bbs-project`, and
+  `--bbs-repo` arguments supplied to the `migrate-repo` command.
```

Comment:
We can make this brief and only mention the visible impact as a feature :ok_hand: 

```suggestion
- Support forked repositories in Bitbucket Server migrations
```